cmake/BinExportDeps.cmake

```diff
@@ -110,6 +115,12 @@ if(BINEXPORT_ENABLE_BINARYNINJA)
   if(NOT binaryninjaapi_POPULATED)
     FetchContent_Populate(binaryninjaapi)  # For binaryninjaapi_SOURCE_DIR
   endif()
+  if(BINEXPORT_BINARYNINJA_LATEST)
+    execute_process(
```

Comment:
Hmm, this won't work on Windows, unfortunately.

Reply:
We should also turn this into a proper dependency. Executing this on every invocation of `cmake --build` is slow.

Alternatively, execute this once per config run.